docs/animation-format.markdown

```diff
@@ -5,7 +5,7 @@ It is far more efficient for a game to store individual parts of an animated spr
 
 Sparrow v2 is the texture atlas format used by Flixel to load and render sprite sheets and animations. It provides a means for a single texture to contain multiple frames, which can later be referenced as needed. When Flixel needs an individual subtexture from a Texture Atlas, it will translate, rotate, scale, and clip the original image before rendering it, rather than loading each individual frame into memory separately. This is convenient to use for animations, but the individual textures can also be referenced as desired.
 
-In Friday Night Funkin' and mods, these sprite sheets are used for many things, including but not limited to sprites for characters and sprites for arrows.
+In Friday Night Funkin' and mods, these sprite sheets are used for pretty much everything except Spirit and the Pixel noteskin. <!-- Why though????????? -->
```

Comment:
I made a modified pixel noteskin that uses the Sparrow atlas ~~and also has 9-key notes~~ so literally the only thing that uses Packer is the Spirit for some reason. That might change in the future too who knows. Also not everything uses spritesheets, like the backgrounds.

- [ ] Revert this line or figure out some more generic wording.